src/theme.tsx

```diff
@@ -0,0 +1,10 @@
+export default {
+  colors: {
+    primary: '#562878',
+    success: 'rgba(23, 150, 23, 0.7)',
+    error: 'rgba(228, 23, 23, 0.75)',
+    disabled: '#a3a3a3',
+    white: '#fff'
+  },
+  font: "'Roboto', sans-serif"
```

Comment:
Also, we should add a notice to readme regarding this font. Example: https://github.com/kiwicom/orbit-components#usage

Reply:
I'm not sure what you mean

Reply:
Just change of readme to notify users about Roboto font. Pushed requested change.